local/local_test.go

```diff
@@ -1591,16 +1598,16 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 				image, err := local.NewImage("not-exist", dockerClient)
 				h.AssertNil(t, err)
 
-				readCloser, err := image.GetLayer("some-layer")
+				readCloser, err := image.GetLayer(someSHA)
 				h.AssertNil(t, readCloser)
-				h.AssertError(t, err, `image "not-exist" does not contain layer with diff ID "some-layer"`)
+				h.AssertError(t, err, fmt.Sprintf("image %q does not contain layer with diff ID %q", "not-exist", someSHA))
 			})
 		})
 	})
 
 	when("#ReuseLayer", func() {
 		var (
-			prevImage     *local.Image
+			prevImage     imgutil.Image
```

Comment:
I think we can change the constructor signature back to returning a pointer, so that this can still be `*local.Image`